examples/notebook/pytorch/Quick_Started_Notebook_of_INC_for_Pytorch.ipynb

```diff
@@ -194,74 +288,40 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### 2.1 Define metric, evaluate function, and dataloader\n",
-    "\n",
-    "In this part, we define a GLUE metirc and use it to generate an evaluate function for INC.\n",
+    "### 2.1 Define calibration function and evaluate function\n",
     "\n",
-    "Refer to doc [metric.md](https://github.com/intel/neural-compressor/blob/master/docs/source/metric.md#build-custom-metric-with-python-api) for how to build your own metric.\n",
-    "Refer to doc [dataset.md](https://github.com/intel/neural-compressor/blob/master/docs/source/dataset.md#user-specific-dataset) and [dataloader.md](https://github.com/intel/neural-compressor/blob/master/docs/source/dataloader.md#build-custom-dataloader-with-python-apiapi) for how to build your own dataset and dataloader."
+    "In this part, we define a GLUE metirc and use it to generate an evaluate function for INC."
```

Comment:
The eval func was not consumed by INC anymore.

Reply:
agree, improved it.